Tevinthuku/Politico
# [Finishes #163670702] Set up the project to work with flask

**What does this PR do?**
This PR sets up the project structure to work with Flask in order to create all the endpoints required

**Task to be completed**
:ballot_box_with_check: Install Flask and Pytest
:ballot_box_with_check: Create home route
:ballot_box_with_check: Set up `GET /offices` route
:ballot_box_with_check: Tested the `/offices` route for status_code response

**How can this be manually tested**
1. Clone this repo & switch to `develop` branch.
2. Visit `/` route
3. Visit `/api/v1/offices` route
4. run `python -m pytest --cov=app/api` to see the test results
